documentation/docs/30-add-ons/25-paraglide.md

```diff
@@ -21,18 +21,18 @@ npx sv add paraglide
 
 ## Options
 
-### availableLanguageTags
+### languageTags
 
 The languages you'd like to support specified as IETF BCP 47 language tags.
 
 ```bash
-npx sv add --paraglide=en,es
+npx sv add paraglide="languageTags:en,es"
 ```
 
 ### demo
 
 Whether to generate an optional demo page showing how to use paraglide.
 
 ```bash
-npx sv add --paraglide=demo
+npx sv add paraglide="demo:yes"
```

Comment:
```suggestion
npx sv add paraglide=demo:yes
```

Reply:
I left these here intentionally since `paraglide` and `tailwindcss` _may_ require quotes depending on what values were specified. 

My line of thinking was this: "could this add-on use a comma at _any_ point?" and if the answer was 'yes', then we'd wrap all of the add-on's values in quotes for the entire docs page

Reply:
Ok, makes sense as well